script.js

```diff
@@ -53,3 +53,20 @@ function handleRadioChange() {
     }
   }
 }
+
+// randomizing the order of the 4 option inputs
+randomizeBtn.addEventListener('click', randomizeAnswers);
+
+function randomizeAnswers() {
```

Comment:
Small "bug" I discovered when randomising each of the rows is that the class style is also randomised.

Right now if I:
1. Enter all the input fields
2. Select the Correct answer
3. Press Random then the styling sticks on the index

Reply:
I fixed it, but maybe there is a simple way to do it. And I had to change the algorithm of sorting to a simple one.